packages/playground/package.json

```diff
@@ -58,6 +58,9 @@
     "prop-types": "^15.7.2",
     "react-app-polyfill": "^1.0.4",
     "react-bootstrap": "^0.33.1",
+    "@fortawesome/free-solid-svg-icons": "^5.13.1",
```

Comment:
Why are these needed? Should we include these as dependencies / peerDependencies for @rjsf/bootstrap-4 and mention them in the readme?

Reply:
we removed this just now. it should be fixed in the latest commit